### DISCUSSION TITLE
[#1040934] New Music Tracks for Editor Mode

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#22] [Feedback]
Right now the default game music plays when your in editor mode, what I'm suggesting is some new music tracks that get the creativity juices flowing.

**Comment by minecraftchloe** [comment]
I adore this idea!

**Comment by CaptainCooky** [comment]
Quite a few Minecraft YT Creators have some skills with Music, maybe Mojang could involve them somehow. Make for a nice gesture to the Community. 😁
